prefect_snowflake/database.py

```diff
@@ -74,3 +77,89 @@ def snowflake_query_flow():
             cursor.get_results_from_sfqid(query_id)
             result = cursor.fetchall()
     return result
+
+
+@task
+async def snowflake_multiquery(
+    queries: List[str],
+    snowflake_credentials: "SnowflakeCredentials",
+    params: Union[Tuple[Any], Dict[str, Any]] = None,
+    cursor_type: Optional[SnowflakeCursor] = SnowflakeCursor,
+    database: Optional[str] = None,
+    warehouse: Optional[str] = None,
+    as_transaction: bool = False,
+    return_transaction_control_results: bool = False,
+) -> List[List[Tuple[Any]]]:
+    """
+    Executes multiple queries against a Snowflake database in a shared session.
+    Allows execution in a transaction.
+
+    Args:
+        queries: The List of queries to execute against the database.
+        params: The params to replace the placeholders in the query.
+        snowflake_credentials: The credentials to use to authenticate.
+        cursor_type: The type of database cursor to use for the query.
+        database: The name of the database to use; overrides
+            the credentials definition if provided.
+        warehouse: The name of the warehouse to use; overrides
+            the credentials definition if provided.
+        as_transaction: If True, queries are executed in a transaction.
+        return_transaction_control_results: Determines if the results of queries
```

Comment:
I am uncertain if this needs to be a keyword; can you provide an example when we would want the BEGIN / COMMIT statements to be returned in the result?

Reply:
They return `Statement executed successfully.` so I imagine that there is a remote possibility that they return something else. Then user code could react to this, but I don't know real life scenario. I also use it in unit tests. Shall If remove this?

Reply:
I think the default to not return the transaction control results is great and the ability to return them if needed is reasonable, so I say we keep it.